package-lock.json

```diff
@@ -1,910 +0,0 @@
-{
```

Comment:
Do we need this file for 'npm release', or can yarn do it?

Reply:
Having two lock files almost certainly doesn't make sense.

I don't think `npm release` should have anything to do with installing packages so I imagine this file is totally unused.

I'm happy to  just update it instead to make the PR more conservative, though.